app/lib/actions/pulp3/content_guard/refresh_all_distributions.rb

```diff
@@ -6,7 +6,8 @@ def plan(smart_proxy)
           sequence do
             plan_action(Actions::Pulp3::ContentGuard::Refresh, smart_proxy)
 
-            roots = ::Katello::RootRepository.where.not(:content_type => ::Katello::Repository::DOCKER_TYPE).where(:unprotected => false)
+            protected_types = [::Katello::Repository::YUM_TYPE, ::Katello::Repository::FILE_TYPE, ::Katello::Repository::DEB_TYPE]
```

Comment:
Is this something we'd want to know outside of this class, or is duplicated anywhere, or alter in the context of this action (given as an input)? ie maybe the protected types should be more centrally located if the answer to either of those is yes

Reply:
I think it makes a lot of sense for each repository type to declare if it supports/makes sense to provide protected content via a content guard (or really if it doesn't make sense.)  That would probably belong more in master rather than 3.18.  I'm happy to just move this and define it somewhere like katello/pulp3/migration.rb or something here if that makes sense? (and file an issue to enhance master)

Reply:
Per standup, let's leave it as-is on this branch. Updating master with what you mentioned sounds like the best way to go, so +1 from me